feat: Deploy Eclipse Che on AKS (#2670)
codenvy/che-docs: modules/administration-guide/pages/installing-che-in-the-cloud.adoc

:_content-type: PROCEDURE

> :description: Installing {prod-short} in the Cloud

Reviewer: ```suggestion
:description: Installing {prod-short} in the cloud
```

:keywords: overview, running-che-in-the-cloud, installing-che-in-the-cloud

> :navtitle: Installing {prod-short} in the Cloud

Reviewer: ```suggestion
:navtitle: Installing {prod-short} in the cloud
```

[id="installing-{prod-id-short}-locally"]

> = Installing {prod-short} in the Cloud

Reviewer: ```suggestion
= Installing {prod-short} in the cloud
```

> This section contains instructions for deploying and running {prod} in the Cloud.

Reviewer: ```suggestion
Deploy and run {prod} in the cloud.
```

.Prerequisites

* A {platforms-name} cluster to deploy {prod-short} on.

> * `{prod-cli}` the command line tool for {prod}. See: xref:installing-the-chectl-management-tool.adoc[].

Reviewer: ```suggestion
* `{prod-cli}`: The command line tool for {prod}. See: xref:installing-the-chectl-management-tool.adoc[].
```

> == Deploying {prod-short} in the Cloud

Reviewer: ```suggestion
== Deploying {prod-short} in the cloud
```

> Choose one of the following procedures to start the {prod-short} Server in the Cloud using the `{prod-cli}` tool.

Reviewer: ```suggestion
Follow the instructions below to start the {prod-short} Server in the cloud using the `{prod-cli}` tool.
```
